test/test_cabin_helper.rb

```diff
@@ -0,0 +1,199 @@
+$: << File.dirname(__FILE__)
+$: << File.join(File.dirname(__FILE__), "..", "lib")
+
+require "rubygems"
+require "minitest-patch"
+require "cabin"
+
+module RSpec
+  class Confi
```

Comment:
Just a hunch that if, in the future, RSpec ended up being loaded somehow I was not clobbering some of RSpec and create a hard to find problem, for the person that included RSpec, later.